pkgs/top-level/python-packages.nix

```diff
@@ -5063,6 +5063,26 @@ in {
     };
   };
 
+  python-jose = buildPythonPackage rec {
```

Comment:
@Mic92 Thanks for your quick review. OK. I've check the new policy, and moved them to /development/python-modules/ directory.